jetty-jmx/src/main/java/org/eclipse/jetty/jmx/ObjectMBean.java

```diff
@@ -235,4 +266,118 @@ MetaData metaData()
             _metaData = MBeanContainer.findMetaData(_mbeanContainer, _managed.getClass());
         return _metaData;
     }
+
+    void notifyGetAttribute(MBeanAttributeInfo attributeInfo, Object value)
+    {
+        listeners().forEach(l ->
+        {
+            try
+            {
+                l.getAttribute(this, attributeInfo, value);
+            }
+            catch (Throwable x)
+            {
+                LOG.info("Failure while invoking listener " + l, x);
+            }
+        });
+    }
+
+    void notifySetAttribute(MBeanAttributeInfo attributeInfo, Object value)
+    {
+        listeners().forEach(l ->
+        {
+            try
+            {
+                l.setAttribute(this, attributeInfo, value);
+            }
+            catch (Throwable x)
+            {
+                LOG.info("Failure while invoking listener " + l, x);
+            }
+        });
+    }
+
+    void notifyInvoke(MBeanOperationInfo operationInfo, Object[] arguments, Object result)
+    {
+        listeners().forEach(l ->
+        {
+            try
+            {
+                l.invoke(this, operationInfo, arguments, result);
+            }
+            catch (Throwable x)
+            {
+                LOG.info("Failure while invoking listener " + l, x);
+            }
+        });
+    }
+
+    private Stream<Listener> listeners()
+    {
+        List<Listener> listeners = _listeners.get();
+        return Stream.concat(_mbeanContainer.listeners().stream(), listeners == null ? Stream.empty() : listeners.stream());
+    }
+
+    @Override
+    public String toString()
+    {
+        return String.format("%s@%x[%s]", getClass().getSimpleName(), hashCode(), getManagedObject());
+    }
+
+    public interface Listener
+    {
+        public default void getAttribute(ObjectMBean objectMBean, MBeanAttributeInfo attributeInfo, Object value)
+        {
+        }
+
+        public default void setAttribute(ObjectMBean objectMBean, MBeanAttributeInfo attributeInfo, Object value)
+        {
+        }
+
+        public default void invoke(ObjectMBean objectMBean, MBeanOperationInfo operationInfo, Object[] arguments, Object result)
```

Comment:
If an invocation throws an exception this method is neither called, nor has the signature to pass the Throwable